examples/cpp/pyperf/PyPerfCollapsedPrinter.cc

```diff
@@ -97,7 +97,7 @@ void PyPerfCollapsedPrinter::processSamples(
           std::fprintf(output_file, ";(missing)_[pe]");
         }
         else {
-          std::fprintf(output_file, ";[Error %d]_[pe]", -stackId);
+          std::fprintf(output_file, ";[Error (errno) %d]_[pe]", -stackId);
```

Comment:
This can just stay "Error %d"

Reply:
I like the mentioning of errno, helps reminding what numbers am I looking at

Reply:
But it's not errno, it's or'ed errnos. So the number as is doesn't necessarily reflect a single errno.

Reply:
`Error (errnos)`? :)